payment-request/payment-response/payerName-attribute-manual.https.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<meta charset="utf8">
+<link rel="help" href="https://w3c.github.io/payment-request/#dom-paymentresponse-payername">
+<title>
+  PaymentResponse.prototype.payerName attribute
+</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="helpers.js"></script>
+<h2>payerName attribute</h2>
+<p>
+  When requested, please use "web platform test" as the payer name.
+</p>
+<ol>
+  <li>
+    <button onclick="runManualTest(this, { requestPayerName: undefined }, { payerName: null })">
+      payerName attribute is null when requestPayerName is undefined.
+    </button>
+  </li>
+  <li>
+    <button onclick="runManualTest(this, { requestPayerName: false }, { payerName: null })">
+      payerName attribute is null when requestPayerName is false.
+    </button>
+  </li>
```

Comment:
Maybe a test for null PaymentOption for test the default browser settings.
button onclick="runManualTest(this, null, { payerName: null })"